photon-client/src/test/scala/com/linkedin/photon/ml/data/avro/AvroDataReaderTest.scala

```diff
@@ -173,7 +173,7 @@ object AvroDataReaderTest {
   private val UnionFieldFloatDouble = "unionFieldFloatDouble"
   private val UnionFloatDoubleValue = 43.5
   private val UnionFieldIntLongFloatDouble = "unionFieldIntLongFloatDouble"
-  private val UnionIntLongFloatDoubleValue = 5
+  private val UnionIntLongFloatDoubleValue = 5.0D
```

Comment:
What was the reason for changing these two values?

Reply:
```  private val avroSchema: Schema = SchemaBuilder
    .record("testAvroSchema")
    .namespace("com.linkedin.photon.ml.avro.data")
    .fields()
    .name(IntField).`type`().intType().noDefault()
    .name(StringField).`type`().stringType().noDefault()
    .name(BooleanField).`type`().booleanType().noDefault()
    .name(DoubleField).`type`().doubleType().noDefault()
    .name(FloatField).`type`().floatType().noDefault()
    .name(LongField).`type`().longType().noDefault()
    .name(MapField).`type`().map().values().stringType().noDefault()
    .name(UnionFieldIntString).`type`().unionOf().intType().and().stringType().endUnion().noDefault()
    .name(UnionFieldIntBoolean).`type`().unionOf().intType().and().booleanType().endUnion().noDefault()
    .name(UnionFieldIntLong).`type`().unionOf().intType().and().longType().endUnion().noDefault()
    .name(UnionFieldFloatDouble).`type`().unionOf().floatType().and().doubleType().and().nullType().endUnion().noDefault()
    .name(UnionFieldIntLongFloatDouble).`type`().unionOf().intType().and().longType().and().floatType().and().doubleType().and().nullType().endUnion().noDefault()
    .name(NullableIntField).`type`().unionOf().intType().and().nullType().endUnion().noDefault()
    .name(NullableBooleanField).`type`().unionOf().booleanType().and().nullType().endUnion().noDefault()
    .name(NullableDoubleField).`type`().unionOf().doubleType().and().nullType().endUnion().noDefault()
    .name(NullableFloatField).`type`().unionOf().floatType().and().nullType().endUnion().noDefault()
    .name(NullableLongField).`type`().unionOf().longType().and().nullType().endUnion().noDefault()
    .name(FeaturesField).`type`().array().items(nameAndTermSchema).noDefault()
    .endRecord()``` In above avro schema, the map expected a string value type; and `UnionIntLongFloatDoubleValue` expect a dominant numeric type which is Double.